### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by kilonzi**
Cloning of MINT.COM signup form

### CODE REVIEW

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Hi there @kilonzi @thykingdoncome 
Nice job on this project team :clap: , you worked a lot on this project. But there somethings that we need to change in order to make in line with Microverse guidelines.

- [x]  It's very important to have a descriptive readme file. This is the first thing that people look in your repository. Don't forget that in the future, your repositories are going to be part of your portfolio. If you need a direction on this you can make something like this one https://github.com/bolah2009/newsweek-clone/blob/master/README.md. The images are optional, but it's best practice have the name and git account of all contributors.

- [x] Your page must have the intuit logo and must be an embedded image that is horizontally centered.

- [x] Your page must contain Mint logo and text (logo on the left, text on the right).

- [x] Your page must contain Quickbooks logo and text (logo on the left, text on the right).

- [x] Your page must contain Turbotax logo and text (logo on the left, text on the right).

- Form

- [x] Must have the title `Create an Intuit account`.

- [x] Must have the description: `One account for everything Intuit, including Mint`.

- [x] Must have a `Learn More` link.

- [x] The input tag for the email must have the attribute `type` with the value `email`.

- [x] The input tag for the phone must have the attribute `type` with the value `tel`.

- [x] The input tag for the password must have the attribute `type` with the value `password`.

- Stickler

- [x] I notice that the linter found some errors in your code, we need to fix those errors. Linting is important to reduce errors and improve the overall quality of your code. Using the linter also gives a professional look in your repository.

Friendly reminder: You should use Github Flow in this project. However, changing it now would take too much time. Please make sure you use Github Flow in the next project. You will find instructions in the Patwright lesson description.

Please kindly make this change and submit for code review again.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+                </h4>
+            </div>
+        </header>
+
+        <section>
+            <div class="form">
+                <form>
+                    <div>
+                        <h2>
+                            Create a Joaze account
```
Comment (Ispirett) [inline index.html]: This should be Create an Intuit account

File: index.html [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+            </div>
+
+        </div>
+
+        <header class="mainHead">
+
+            <div class="title">
+                <h1>
+                    <a href="#">
+                        J O A Z E
```
Comment (Ispirett) [inline index.html]: This should Intuit

File: index.html [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+        <section>
+            <div class="form">
+                <form>
+                    <div>
+                        <h2>
+                            Create a Joaze account
+                        </h2>
+
+                        <p>
+                            One account for everything Joaze. <a href="#">Learn more</a>
```
Comment (Ispirett) [inline index.html]: This should be One account for everything Intuit, including Mint

File: index.html [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+            <div class="title">
+                <h1>
+                    <a href="#">
+                        J O A Z E
+                    </a>
+                </h1>
+            </div>
+
+            <div class="c">
```
Comment (Ispirett) [inline index.html]: This section should have the original should have the icon text like the original page.

#### Make sure
 - [x] It shows the Mint logo and text (logo on the left, text on the right).

 - [x] It shows the Quickbooks logo and text (logo on the left, text on the right).

 - [x] It shows the Turbotax logo and text (logo on the left, text on the right).

**Review by Ispirett — CHANGES_REQUESTED** [review]
Hey @kilonzi  @thykingdoncome   Great! job implementing some of the changes suggested by @Torres-ssf 
image. 

#### But there are still some things to address 

- [x]  There is an issue to address with stickler
- [x] You can add the creators of the project to your readme file

Friendly reminder: You should use Github/Git Flow in this project. However, changing it now would take too much time.
Please make sure you use Github/Git Flow in the next project. You will find instructions in the Patwright lesson description.

File: index.html (hunk 1 above)
Comment (kilonzi) [reply index.html]: Resolved

File: index.html (hunk 4 above)
Comment (kilonzi) [reply index.html]: We realised that in the original page, these where images and we have  now included them

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

**Comment by Torres-ssf** [comment]
### Way to go @kilonzi @thykingdoncome :clap: 
Great job making these changes. I'm giving this projet a :+1:. You can merge to master now. Good luck with the nex one :smiley: 

And just a friendly reminder :smile: 
Please make sure you use Github Flow in the next project. You will find instructions in the Patwright lesson description.

**Review by Torres-ssf — APPROVED** [review]
### Way to go @kilonzi @thykingdoncome :clap: 
Great job making these changes. I'm giving this projet a :+1: . You can merge to master now. Good luck with the nex one smiley

And just a friendly reminder smile
Please make sure you use Github Flow in the next project. You will find instructions in the Patwright lesson description.

**Pull request merged**